### ISSUE TITLE
cors

### ISSUE THREAD

**Issue by marchenko1985** [grahamjenson/hapiger#16]
will be nice if cors will work out of the box or at least with some command line flag

**Comment by grahamjenson** [comment]
Accepting PRs always :)

**Comment by pbojinov** [comment]
Edit: Never mind, noticed his was not released, hence the errors still.

When testing this with a client-side app, noticed that there is no support for the CORS preflight request.

![image](https://user-images.githubusercontent.com/709616/29303697-84cd9f88-8143-11e7-9bbe-1e80e9878934.png)

**Comment by marchenko1985** [comment]
Yep for this to work just add line from PR and everything will start to work as expected

Also take attention that it won't work under heavy load but this is another story
